src/utils/YotiPayloadEnums.ts

```diff
@@ -44,7 +44,7 @@ export const YOTI_CHECKS = {
 		"type": "DOCUMENT_SCHEME_VALIDITY_CHECK",
 		"config": {
 			"manual_check": MANUAL_CHECK_TYPE.IBV,
-			"scheme": "UK_DBS",
+			"scheme": "UK_GDS",
```

Comment:
could this be moved to Constants

Reply:
This file is being used as Constants for the Yoti payload generation logic, did you want me to take out "UK_GDS" and move the string to /src/utils/Constants.ts?

Reply:
ah okay seems fine then

Reply:
Moved to a constant within the same file for clarity :)